### PR TITLE
Holorecord fixes and additions

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
add: You can now rename and copy holodisks!
add: Holorecordings now can be looped.
tweak: Holodisks now have materials and the autolathe cost has been adjusted.
fix: Fixed holodisk speaker name not getting set when on non-prerecorded messages.
/:cl:

[why]: Fixes holorecords not setting speaker name. Checked for Unknown, since that's the default and it's required for the preprocessed disks. 
Removed dots from the buttons because it looks weird.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
nani tf are holodisks

**Review by AnturK — DISMISSED** [review]
Looks good.

**Comment by Anonmare** [comment]
I use them to anonymously insult people

**Comment by DaxDupont** [comment]
I made it so you can set them to loop, so people can have advertisement holopads and what not. Figured that was cool to do.

File: code/datums/holocall.dm [hunk 1]
```diff
@@ -220,6 +222,14 @@
 	QDEL_NULL(record)
 	return ..()
 
+/obj/item/disk/holodisk/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/disk/holodisk))
+		var/obj/item/disk/holodisk/holodiskOriginal = W
+		record = holodiskOriginal.record
```
Comment (AnturK) [inline code/datums/holocall.dm]: This should probably copy it not reference, cleaning the disk deletes the record.

**Review by AnturK — CHANGES_REQUESTED** [review]
The rest looks good.

**Pull request merged**